docs/client/models.md

```diff
@@ -0,0 +1,50 @@
+# <img align="center" src="../images/logo.png">  Accessing Models and Enums
+
+## General
+
+Models and enums are generated in the `models` namespace. So, say you are using package `azure.pets`. To access model `Dog`, you would use the following code
+snippet
```

Comment:
Why is this on a separate line?

Reply:
I think I just did it to make it cleaner looking at the raw markdown file. Doesn't render any differently in actual markdown, but I'll get rid of the separate line bc it looks a little weird